upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.12.yaml.template

```diff
@@ -606,7 +800,7 @@ spec:
             # chosen from this range. Changing this value after installation will have
             # no effect. This should fall within `--cluster-cidr`.
             - name: CALICO_IPV4POOL_CIDR
-              value: "{{ .KubeControllerManager.ClusterCIDR }}"
```

Comment:
I'm not sure whether this causes issues if the value of `ClusterCIDR` is different to `192.168.0.0/16` - my intuition is that it would.

Reply:
Great catch, thank you!